libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -612,6 +614,42 @@ public synchronized void setBrowserLoginEnabled(boolean browserLoginEnabled) {
     }
 
     /**
+     * Returns whether the IDP login flow is enabled.
+     *
+     * @return True - if IDP login flow is enabled, False - otherwise.
+     */
+    public boolean isIDPLoginFlowEnabled() {
```

Comment:
This enables IDP login flow within the SP. The visibility of the button that triggers this flow in `LoginActivity` depends on this flag too.